cvat-ui/src/utils/opencv-wrapper/tracker-mil.ts

```diff
@@ -0,0 +1,51 @@
+// Copyright (C) 2022 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import { TrackerModel, TrackingResult } from './opencv-interfaces';
+
+export type TrackerMIL = TrackerModel;
+
+export default class TrackerMILImplementation implements TrackerMIL {
+    public name: string;
+    private prevImageData: ImageData | null;
```

Comment:
I would suggest to rename to ``imageData``  because of the way the identificator is used.

Reply:
Done